local/rest_api_gcbm/Dockerfile

```diff
@@ -10,9 +10,15 @@ RUN pip install --no-cache-dir -r requirements.txt
 # Adding npm for CML Actions
 RUN apt-get update 
 RUN apt-get install nodejs npm -y
+RUN apt-get install -y python3-dev \
+    && cd /usr/local/bin \
+    && ln -s /usr/bin/python3 python
+RUN python --version
 RUN node --version
 RUN npm --version
 
+RUN git clone https://github.com/moja-global/GCBM.CompileResults.git
```

Comment:
Not sure if installing it here is intuitive. Can you make a seperate Dockerfile for your purpose?

Reply:
Should I put it in `docker\Dockerfile.gcbm` ?